client/src/components/geneExpression/quickGene.tsx

```diff
@@ -50,14 +51,27 @@ function QuickGene() {
     fetch();
 
     async function fetch() {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on commit
       if (annoMatrix !== (prevProps as any)?.annoMatrix) {
         const { schema } = annoMatrix;
         const varIndex = schema.annotations.var.index;
 
         setStatus("pending");
         try {
           const df: Dataframe = await annoMatrix.fetch("var", varIndex);
-          setStatus("success");
+          let dfIds: Dataframe;
+          const geneIdCol = "feature_id";
+
+          // if feature id column is available in var
+          if (annoMatrix.getMatrixColumns("var").includes(geneIdCol)) {
+            dfIds = await annoMatrix.fetch("var", geneIdCol);
+            console.log("id, success");
+            setGeneIds(dfIds.col("feature_id").asArray() as DataframeValue[]);
+          } else {
+            console.error("Could not find feature IDs.");
+          }
+
+          setStatus("name, success");
```

Comment:
why the change, just curious

Reply:
This code is only for datasets that already have the "feature_id" column. Otherwise, we'll call the API with only gene name, since we don't have gene IDs built into the dataset (more of the code is implemented in the other PR that just recently got merged to this one)